devbox.json

```diff
@@ -13,20 +12,25 @@
     "envsubst@latest",
     "mockgen@latest",
     "jq@latest",
-    "openssl@latest"
+    "openssl@latest",
+    "go@1.22.5"
   ],
   "shell": {
     "init_hook": [
-      "export GOROOT=$(go env GOROOT)"
+      "echo 'Welcome to devbox!' > /dev/null"
     ],
     "scripts": {
-      "local-deploy": "make local-deploy",
+      "local-deploy":          "make local-deploy",
       "remote-cluster-deploy": "make remote-cluster-deploy",
-      "e2e-test": "make e2e-test",
-      "cleanup-cluster": "make cleanup-cluster"
+      "e2e-test":              "make e2e-test",
+      "cleanup-cluster":       "make cleanup-cluster",
+      "csi-sanity-test":       "make csi-sanity-test"
```

Comment:
Since this is a json file, I wonder if we should do such formatting. Some editors/tools might just revert these white spaces.

Reply:
Yeah I think vscode automatically added the whitespaces for me. I'll remove those